packages/react/src/components/DataTable/stories/DataTable-ai-label.stories.js

```diff
@@ -63,10 +65,49 @@ const columnAILabelHeaders = [
   {
     key: 'port',
     header: 'Port',
+    decorator: (
+      <AILabel
+        className="ai-label-container"
+        autoAlign={false}
+        align="bottom-right">
+        <AILabelContent>
+          <div>
+            <p className="secondary">AI Explained</p>
+            <h1>84%</h1>
+            <p className="secondary bold">Confidence score</p>
+            <p className="secondary">
+              Lorem ipsum dolor sit amet, di os consectetur adipiscing elit, sed
+              do eiusmod tempor incididunt ut fsil labore et dolore magna
+              aliqua.
+            </p>
+            <hr />
+            <p className="secondary">Model type</p>
+            <p className="bold">Foundation model</p>
+          </div>
+          <AILabelActions>
+            <IconButton kind="ghost" label="View">
+              <View />
+            </IconButton>
+            <IconButton kind="ghost" label="Open Folder">
+              <FolderOpen />
+            </IconButton>
+            <IconButton kind="ghost" label="Folders">
+              <Folders />
+            </IconButton>
+            <Button>View details</Button>
+          </AILabelActions>
+        </AILabelContent>
+      </AILabel>
+    ),
   },
   {
     key: 'rule',
     header: 'Rule',
+    decorator: (
+      <Tooltip>
+        <Information />
+      </Tooltip>
```

Comment:
This is non-blocking, but it'd be nice if all these "info" tooltips had sample content in them, but they do render as expected as is.

Reply:
Thanks for the suggestion, added text as well.